File: app/src/main/java/id/go/patikab/rsud/remun/remunerasi/model/object/LoginResponse.java
package id.go.patikab.rsud.remun.remunerasi.model.object;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class LoginResponse {
    @SerializedName("status")
    String status;
    @SerializedName("message")
    String message;
    @SerializedName("data")
    List<DataUser> dataUser;

    public String getStatus() {
        return status;
    }

    public String getMessage() {
        return message;
    }

    public List<DataUser> getDataUser() {
        return dataUser;
    }
}

File: app/src/main/java/id/go/patikab/rsud/remun/remunerasi/data/api/objectResponse/LoginResponse.kt
package id.go.patikab.rsud.remun.remunerasi.data.api.objectResponse

import com.google.gson.annotations.SerializedName
import java.security.Signature

data class LoginResponse(@SerializedName("status") val status: String? = null,
                         @SerializedName("message") val message: String? = null,
                         @SerializedName("data") val dataUser: List<Akun>? = null) {
    data class Akun(@SerializedName("username") val username: String? = null,
                    @SerializedName("status") val status: String? = null,
                    @SerializedName("kd_dokter") val kd_dokter: String? = null,
                    @SerializedName("email") val email_user: String? = null,
                    @SerializedName("nm_dokter") val nama_dokter: String? = null,
                    @SerializedName("signature") val signatured: String? = null)
}
